docs/deployment/migration.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Migration
+parent: Deployment
+nav_order: 7
+---
+
+# Migrating from Google Santa to NPS Santa
```

Comment:
Nit: We never call out what NPS is in the doc.

Maybe:
`# Migrating to North Pole Security (NPS) Santa from Google Santa`

But this might be long for a large font title... If that doesn't fit, I think it'd be good to call this out in the `Overview`

Reply:
Reasonable to call out in the `Overview`, also we can call it out on the main doc page as well.